plugins/org.python.pydev.parser/src/org/python/pydev/parser/grammar38/python.jjt

```diff
@@ -421,11 +421,19 @@ MORE : /* Strings */
 |   < (["b", "B"]) (["r", "R"])? "\"" > :  IN_BSTRING21
 |   < (["b", "B"]) (["r", "R"])? "'''" > :  IN_BSTRING13
 |   < (["b", "B"]) (["r", "R"])? "\"\"\"" > :  IN_BSTRING23
+|   < (["r", "R"] ["b", "B"])? "'" > :  IN_BSTRING11
```

Comment:
Seems good, the only thing is that these changes now have to be ported to python.jjt_template (and then, run make_replace.py to update this file).